docker/README.md

```diff
@@ -1,3 +1,28 @@
+# Build emacs-ng inside a Docker container
```

Comment:
I think we should put the docs to the readme folder.

Reply:
you mean moving this part into `./docs` (the web documntation), correct?

(I agree)